docs/how_to/install.md

```diff
@@ -75,3 +76,14 @@ There are several ways to install the package, in all these cases the TVM librar
     #       providing --user flag may trigger error during installation in such case.
     cd python; python setup.py install --user; cd ..
     ```
+
+## Solution to Installation Error
+
+If you encounter the problem while installation process, you can solve by updating submodules to the latest commit set.
+To update submodules to the latest commit set, type the following command.
+
+```bash
+git submodule foreach git pull origin master
```

Comment:
change to git submodule update --init --recursive

Reply:
Fixed it. please check cb2eaf3